config/default/migrate_plus.migration.hs_capx.yml

```diff
@@ -297,9 +297,12 @@ process:
       plugin: flatten
   department:
     -
```

Comment:
Should keep the `skip_on_empty` to prevent any errors.

Reply:
I actually was wondering about this as it relates to the `null_coalesce` process.

From my understanding, `null_coalesce` will run through the sources values specified and pick the first one that's not `NULL`. So in this migration, I assumed it was meant to use the `org_title` value as the Department if the `department` value was `NULL`. However, because there's a `skip_on_empty` above the `null_coalesce` wouldn't it never fire/work? Because if the department value is empty, it will skip it and never get to the `null_coalesce` process.